README.md

```diff
@@ -5,8 +5,15 @@
 [![Code Climate](https://codeclimate.com/github/ocadotechnology/rapid-router/badges/gpa.svg)](https://codeclimate.com/github/ocadotechnology/rapid-router)
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/code-for-life/localized.svg)](https://crowdin.com/project/code-for-life)
 
-Rapid Router is a 'Code For Life' project, aimed at teaching children programming concepts through a vehicle routing
-game.
+## A  [Code for Life](https://www.codeforlife.education/) project
+* Rapid Router is a [Code for Life](https://www.codeforlife.education/) project, aimed at teaching children programming concepts through a vehicle routing
+  game. 
+  Ocado Technology's [Code for Life initiative](https://www.codeforlife.education/) has been developed to inspire the next generation of computer scientists and to help teachers deliver the computing curriculum.
```

Comment:
Maybe we should mention that "the computing curriculum" is part of the national curriculum as this is quite new.